controllers/recipes.js

```diff
@@ -199,16 +199,20 @@ module.exports = {
     }
   },
   deleteRecipe: async (req, res) => {
+    console.log('you clicked the trashcan')
```

Comment:
I changed up the logging of this block just a bit when testing locally, if you wanna take a look and consider implementing it. I do think at some point we should go through everything and start standardizing how and where things are logged.

```
deleteRecipe: async (req, res) => {
  console.log('deleteRecipe was invoked')
  try {
    const recipeDbId = req.params.id;
    console.log("   Finding recipe id: ", recipeDbId);
    const recipe = await Recipe.findById({ _id: recipeDbId });
    const recipeCloudinaryId = recipe.cloudinaryId;
    console.log("   Found CloudinaryID on recipe: ", recipeCloudinaryId);
    await cloudinary.uploader.destroy(recipeCloudinaryId);
    console.log("   Deleted CloudinaryID image: ", recipeCloudinaryId);
    await recipe.remove({ _id: recipeDbId });
    console.log("   Deleted recipe ID from db: ", recipeDbId);
    console.log("✅ deleteRecipe has completed successfully");
    res.redirect("/profile");
  } catch (err) {
    console.log("An error has occurred while attempting to delete a recipe.");
    console.log(err);
    res.redirect("/profile");
  }
},
```

Reply:
I'm just leaving this as a comment and not a "request". I'll leave it up to you if you want to replace it with what I have here. It's not critical to the PR, just enhances the logging a tiny bit.

Reply:
it was added. The logs are great.